index.html

```diff
@@ -6,15 +6,124 @@
+            <a href="#projects">projects.</a>
+        </li>
+        <li class="sm-col-12 md-col-4 lg-col-4">
+            <a href="#about">about.</a>
+        </li>
+        <li class="sm-col-12 md-col-4 lg-col-4">
+            <a href="#contact">contact.</a>
+        </li>
+    </ul>
+    
+</header>
+     
+<main>
+<section class="header-hero">
+    
+</section>
+<section class="container hero-content">
+<h3>projects.</h3>
 
-</body>
+<div class="row">
+    <div class="sm-col-12 md-col-12 lg-col-4">
+        <img class= "hero-content-img"src="https://images.unsplash.com/photo-1492144534655-ae79c964c9d7?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=847&q=80" alt="">
+    </div>
+    <div class="sm-col-12 md-col-12 lg-col-4">
+        <img class= "hero-content-img" src="https://images.unsplash.com/photo-1513036191774-b2badb8fcb76?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60" alt="">
+    </div>
+    <div class="sm-col-12 md-col-12 lg-col-4">
+        <img class= "hero-content-img" src="https://images.unsplash.com/photo-1525609004556-c46c7d6cf023?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=389&q=80" alt="">
+    </div>
+</div>
+
+<div class="row">  
+    <div class="sm-col-12 md-col-12 lg-col-4">
+        <img class= "hero-content-img" src="https://images.unsplash.com/photo-1514316454349-750a7fd3da3a?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60" alt="">
+    </div>
+    <div class="col-6 sm-col-6 md-col-4 lg-col-4 xl-col-3">
+        <img class= "hero-content-img" src="https://images.unsplash.com/photo-1550561438-3c2c3b8ea3a6?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60" alt="">
+    </div>
+    <div class="col-6 sm-col-6 md-col-4 lg-col-4 xl-col-3">
+        <img class= "hero-content-img" src="https://images.unsplash.com/photo-1520031441872-265e4ff70366?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60" alt="">
+    </div>
+</div>
 
+<div class="row">  
+    <div class="sm-col-12 md-col-12 lg-col-4">
+        <img class= "hero-content-img" src="https://images.unsplash.com/photo-1550855909-c4f6798dd77f?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=750&q=80" alt="">
+    </div>
+    <div class="sm-col-12 md-col-12 lg-col-4">
+        <img class= "hero-content-img" src="https://images.unsplash.com/photo-1574904735613-d27b0559232c?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=500&q=60" alt="">
+    </div>
+    <div class="sm-col-12 md-col-12 lg-col-4">
+        <img class= "hero-content-img" src="https://images.unsplash.com/photo-1547757015-025b1a633637?ixlib=rb-1.2.1&ixid=eyJhcHBfaWQiOjEyMDd9&auto=format&fit=crop&w=1534&q=80" alt="">
+    </div>
+</div>
+    
+</section>
+<div class="before-about-img"></div>
+
+<section class="container about">
+    <div class="row hero-content-row">
+        
+        <div class="sm-col-12 md-col-12 lg-col-12">
```

Comment:
If you're setting the same number of columns for all breakpoints, you don't need to use a class for each one (for example `col-12 sm-col-12 md-col-12 lg-col-12`). You only need a column class for different sizes if you're _changing_ the number of columns for different sizes, for example `col-12 md-col-6 lg-col-4`. 
So in this case just using `col-12` will be enough.